release-notes/6.0/preview/api-diff/preview7/WindowsDesktop/6.0-preview7.md

```diff
@@ -0,0 +1,30 @@
+# API Difference 5.0.0 vs 6.0.0-preview.7.21378.9
+
+API listing follows standard diff formatting. Lines preceded by a '+' are
+additions and a '-' indicates removal.
+
+* [Microsoft.Win32](6.0-preview7_Microsoft.Win32.md)
```

Comment:
For any of these where we exposed the types in the base shared framework, we should remove them from the diff.

Reply:
In other words, most of the diffs which show up as completely red aren't actually removals because the assembly was just pushed down.